web/js/codeworld_shared.js

```diff
@@ -1541,11 +1367,17 @@ function preFormatMessage(msg) {
   msg = Html.encode(msg)
     .replace(
       /program\.hs:(\d+):((\d+)(-\d+)?)/g,
-      '<a href="#" onclick="goto($1, $3); return false;">Line $1, Column $2</a>'
+      `<a href="#" onclick=${() => {
```

Comment:
Can you explain how this works?  I'm confused.  We're building a string here, not a DOM element, so how can you embed a literal function into it?

Reply:
I got mixed up, interpolating a function here wouldn't work. Will revert this change.